mobile/src/main/java/org/openhab/habdroid/background/ItemsControlsProviderService.kt

```diff
@@ -201,6 +206,30 @@ class ItemsControlsProviderService : ControlsProviderService() {
         }
     }
 
+    private fun getItemLocation(item: Item, allItems: List<Item>): String? {
+        val groups = mutableListOf<Item>()
+        item.groupNames.forEach { groupName ->
+            val group = allItems.first { item -> item.name == groupName }
+            // Check if item is in a location group
+            val isLocation = group.tags.any { tag -> tag in LOCATION_TAGS }
+            if (isLocation) {
+                return group.label
```

Comment:
I'm not sure if we should show the label of the group tagged with a location tag or if we should translate the tag to a string.

Reply:
I'd use group label for `Item.Tag.Location` and a localized name in case of the specialized tags.

Reply:
Sounds like a good idea. I guess we have to add a section to the docs on how the subtitle is generated.